plugins/pci-psp/fu-pci-psp-device.c

```diff
@@ -83,8 +83,10 @@ fu_pci_psp_device_set_valid_data(FuDevice *device, FuSecurityAttrs *attrs)
 	attr = fu_security_attrs_get_by_appstream_id(attrs,
 						     FWUPD_SECURITY_ATTR_ID_SUPPORTED_CPU,
 						     NULL);
-	if (attr != NULL)
+	if (attr != NULL) {
```

Comment:
Do you think setting success should always remove this flag? I.e. move the action to the security attr object

Reply:
fixed